.github/workflows/code-examples.yml

```diff
@@ -0,0 +1,49 @@
+name: Code examples version and deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      hotVersion:
+        description: 'Handsontable version to update all code examples (eg.: 8.0.0)'
+
+jobs:
+  Examples:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # https://github.com/actions/checkout/releases/tag/v2.3.4
+      - name: Use Node 14
+        uses: actions/setup-node@c46424eee26de4078d34105d3de3cc4992202b1e # https://github.com/actions/setup-node/releases/tag/v2.1.4
+        with:
+          node-version: '14'
+      - name: Checkout gh-pages branch to "./examples/tmp" folder
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+          path: './examples/tmp'
+      - name: Install global dependencies
+        run: npm install
+      - name: Create new examples for version
+        run: npm run examples:version ${{ github.event.inputs.hotVersion }}
+        # this action can throw an error, when the examples for specific version exist already. If they exist, move to the next action
+        continue-on-error: true
```

Comment:
It's hard to achieve with two separate jobs. The first job `Version` would create a folder with new code examples and the second job `Deploy` would perform its own actions on that new examples. But how to transfer these newly created examples from one job to another? GH Actions doesn't see the new files created in the `Version` job in the `Deploy` job. Even if I commit them just after creation, they still aren't present in the second job. I can see the solution of moving them as an artifact to the second job, but there is one downside of this approach (despite more complexity) - in case of failure of the first job, the whole workflow is marked as failed, even if the second job passed.

If I missed something please update @budnix @jansiegel